packages/protvista-structure/src/protvista-structure.js

```diff
@@ -129,7 +135,11 @@ class ProtvistaStructure extends HTMLElement {
     this.appendChild(this.messageContainer);
     this.appendChild(flexContainer);
     flexContainer.appendChild(litemolDiv);
-    flexContainer.appendChild(this.tableDiv);
+    if (!this.hideTable) {
+      this.tableDiv = document.createElement("div");
+      this.tableDiv.className = "table-container";
+      flexContainer.appendChild(this.tableDiv);
+    }
     this.loadLiteMol();
     this.loadUniProtEntry().then(entry => {
```

Comment:
Can we bypass this if we don't show the table? It adds an unnecessary call to the UniProt API.